utils/utils.go

```diff
@@ -5,13 +5,45 @@ import (
 	"encoding/json"
 	"io/ioutil"
 	"os"
+	"strconv"
 
 	"golang.org/x/oauth2"
 	"golang.org/x/oauth2/google"
 	"golang.org/x/oauth2/jwt"
 	admin "google.golang.org/api/admin/directory/v1"
 )
 
+
+// type Foo struct {
+//   Value     FlexInt
+// }
+
+// A FlexInt is an int that can be unmarshalled from a JSON field
+// that has either a number or a string value.
+// E.g. if the json field contains an string "42", the
+// FlexInt value will be "42".
+type FlexInt int
+
+// UnmarshalJSON implements the json.Unmarshaler interface, which
+// allows us to ingest values of any json type as an int and run our custom conversion
+
+func (fi *FlexInt) UnmarshalJSON(b []byte) error {
+   if b[0] != '"' {
```

Comment:
if the first byte of the field is not a `"` than it should be an int and casts it directly. @benwaffle